documentation/feature/spectator.md

```diff
@@ -0,0 +1,356 @@
+
+Note that the unit whitelist/blacklist also serves to override this side filtering mechanism.
+
+The default side list is `[west,east,resistance,civilian]` and to update it (on the local client) function `ace_spectator_fnc_updateSpectatableSides` can be used:
+```
+ * Arguments:
+ * 0: Sides to add <ARRAY>
+ * 1: Sides to remove <ARRAY>
+ *
+ * Return Value:
+ * Spectatable sides <ARRAY>
+ *
+ * Example:
+ * [[west], [east,civilian]] call ace_spectator_fnc_updateSpectatableSides
+```
+
+The side filter determines which sides from the side list are valid each time the unit list is updated. It's controlled by setting `ace_spectator_filterSides` and there are four possible options:
+- Player side (default)
+- Friendly sides
+- Hostile sides
+- All sides
+
+### 1.4 Camera Modes
+
+There are 3 possible camera modes:
+- Free
+- Internal
+- External
+
+Scenario designers can control the camera modes available to spectators via the setting `ace_spectator_restrictModes`. Function `ace_spectator_fnc_updateCameraModes` is also provided to alter the available modes (to the local player) as desired at any point in the mission:
+```
+ * Arguments:
+ * 0: Camera modes to add <ARRAY>
+ * 1: Camera modes to remove <ARRAY>
+ *
+ * Return Value:
+ * Available camera modes <ARRAY>
+ *
+ * Example:
+ * [[0], [1,2]] call ace_spectator_fnc_updateCameraModes
+```
+
+### 1.5 Vision Modes
+
+Vision modes are only available in free camera mode. By default there are 4 available vision modes:
+- Normal
+- Night vision
+- Thermal imaging (white hot)
+- Thermal imaging (black hot)
+
+Scenario designers can control which of these vision modes are available to spectators via the setting `ace_spectator_restrictVisions`. However, there are actually a total of 10 possible vision modes and function `ace_spectator_fnc_updateVisionModes` can be used to alter which of them are available (to the local player) at any point in the mission:
+```
+ * Possible vision modes are:
+ *   - -2: Normal
+ *   - -1: Night vision
+ *   -  0: White hot
+ *   -  1: Black hot
+ *   -  2: Light Green Hot / Darker Green cold
+ *   -  3: Black Hot / Darker Green cold
+ *   -  4: Light Red Hot / Darker Red Cold
+ *   -  5: Black Hot / Darker Red Cold
+ *   -  6: White Hot / Darker Red Cold
+ *   -  7: Thermal (Shade of Red and Green, Bodies are white)
+ *
+ * Arguments:
+ * 0: Vision modes to add <ARRAY>
+ * 1: Vision modes to remove <ARRAY>
+ *
+ * Return Value:
+ * Available vision modes <ARRAY>
+ *
+ * Example:
+ * [[0], [1,2]] call ace_spectator_fnc_updateVisionModes
+```
+
+### 1.6 Camera Attributes
+
+The spectator camera has 8 manipulatable attributes:
+- Camera mode: The camera view
```

Comment:
a little bit of text formatting would be awesome here. Maybe have the options "bold" and the description "normal". Or options "normal" and description "italic".